.github/workflows/test.yml

```diff
@@ -3,7 +3,6 @@ name: CI
 on:
   pull_request:
   push:
-    branches: [main]
```

Comment:
Removed this to see the result on the CI on my fork: https://github.com/browniebroke/drf-excel/actions

GitHub actions don't run new PR workflow added from a forks (I think)